src/pspm_convert_pixel2unit.m

```diff
@@ -135,11 +135,11 @@
         
         % convert according to range
         chan.data = (chan.data-chan.header.range(1)) ...
-            ./ diff(chan.header.range) * fact;
+                    * fact ./ (diff(chan.header.range) + 1);
```

Comment:
Not sure why we have `+ 1` now? Because of a possible division by 0? If so, that should be handled by a separate if check instead of changing the result of the operation.

Reply:
This is because the `chan.header.range` is a range of pixel which starts from 0, e.g. `[0,1023]` so there is in total 1024 pixels. Shall I specify it in a comment ?

Reply:
As far as I know, if the range is `[0, 1023]` then the pixel values are also in `[0, 1023]`. So an extra `+1` in the denominator would change the resulting ratio. For example, it can never be `1` anymore, meaning we would get slightly smaller results in the target unit.